lib/cassieq/client/messages.rb

```diff
@@ -3,16 +3,18 @@
 module Cassieq
   class Client
     module Messages
-      def publish_message(queue_name, message)
-        request(:post, "queues/#{queue_name}/messages", message)
+      def publish_message(queue_name, message, initial_invis_time = nil)
+        query = { initialInvisibilityTime: initial_invis_time } unless initial_invis_time.nil?
```

Comment:
No worries. In general its always good to include the unit type if the data type itself isn't expressive. This was a bad decision on our part to not include the time unit in the name originally